dateparser/conf.py

```diff
@@ -77,11 +81,15 @@ def replace(self, mod_settings=None, **kwds):
 def apply_settings(f):
     @wraps(f)
     def wrapper(*args, **kwargs):
-        mod_settings = kwargs.get('settings')
-        kwargs['settings'] = mod_settings or settings
+        mod_settings = kwargs.get('settings', {})
+        if mod_settings is None:
+            kwargs['settings'] = mod_settings = {}
```

Comment:
I would avoid this double assignment. You could do something like this:

```suggestion
            kwargs['settings'], mod_settings = {}, {}
```

There are multiple reasons, but basically to ease the code review. You can find more context here: https://stackoverflow.com/questions/4172223/double-assignment-should-it-be-avoided

Reply:
Sure, done.